packages/adapter-node/src/polka-handler.js

```diff
@@ -15,7 +15,7 @@ const paths = {
 	prerendered: join(__dirname, '/prerendered')
 };
 
-export function createServer({ render }) {
+export function createPolkaHandler({ render }) {
```

Comment:
would end users call this? If not, we should use underscores and call it `create_polka_handler`